src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -311,8 +311,8 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    * responsibility to deal with transient connection errors (if
    * applicable) once initialized.
    *
-   * The behavior of calling initialize() on an already initialized
-   * instance should be a no-op, but if an error was raised
+   * If called on an already initialized instance, re-initialize with
+   * any updated settings that are provided. If an error was raised
    * previously, then initialization should be re-attempted.
```

Comment:
[Question] Does mentioning that we redo initialization if an error was raised previously still parse? Seems like this paragraph is saying "Reinitialize, if an error was raised ... then reinitialize." Which seems like a redundant statement.

Reply:
I think we need to disambiguate between re-initializing with updated settings, vs. retrying a failed initialization even if no settings have changed.  Can you think of a better wording for that?

The corresponding `Manager` docs say
```
   * The behavior of calling initialize() on an already initialized
   * Manager is to re-initialize the manager with any updated settings
   * that are provided. If an error was raised previously, then
   * initialization will be re-attempted.
```

Reply:
I see, I guess the current wording parses then.